### PR TITLE
Mobile path

### PR DESCRIPTION
**Pull request by ghost**
`/m` path to solve #6353.

### CODE REVIEW

**Comment by jhass** [comment]
I think I prefer `/m` idea, it doesn't require any additional setup changes.

**Comment by jhass** [comment]
Also this doesn't work if the main host of your pod is already second level.

**Comment by jhass** [comment]
Oh and as I said I think it should redirect to `stream_path`.

**Comment by AugierLe42e** [comment]
Looks better ?

File: config/routes.rb [hunk 1]
```diff
@@ -18,6 +18,8 @@
     mount Sidekiq::Web => '/sidekiq', :as => 'sidekiq'
   end
 
+  get "/m", to: "home#force_mobile"
```
Comment (jhass) [inline config/routes.rb]: Let's move that down to `mobile/toggle`

File: features/mobile/home.feature [hunk 2]
```diff
@@ -10,3 +10,10 @@ Feature: Visit the landing page of the pod
     When I toggle the mobile view
     And I go to the root page
     Then I should see "Welcome, friend"
+
+    When I am on the root page
+    Then I should see "Welcome, friend"
+    When I go to the mobile path
+    Then I should see "LOG IN"
+    And I go to the mobile path
```
Comment (jhass) [inline features/mobile/home.feature]: `When`

File: features/mobile/home.feature [hunk 3]
```diff
@@ -10,3 +10,10 @@ Feature: Visit the landing page of the pod
...
+    And I should see "LOG IN"
```
Comment (jhass) [inline features/mobile/home.feature]: `Then`

File: config/routes.rb (hunk 1 above)
Comment (jhass) [inline config/routes.rb]: Also a prettier name would be good, how about `as: `force_mobile`?

**Comment by AugierLe42e** [comment]
I think this is ready for merge.

**Comment by jhass** [comment]
Merged as 2ce7a1e185eeda9cb0b87064febb0e228e65264b

Thanks.

**Comment by Flaburgan** [comment]
Thanks! Will be useful for https://github.com/Flaburgan/diaspora-webapp

**Pull request closed without merging**